src/Components/Common/MarkdownPreview.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import ReactMarkdown from "react-markdown";
+import rehypeRaw from "rehype-raw";
+import { UserModel } from "../Users/models";
+import useQuery from "../../Utils/request/useQuery";
+import routes from "../../Redux/api";
+
+interface UserCardProps {
+  user: UserModel;
+}
+
+const UserCard: React.FC<UserCardProps> = ({ user }) => (
+  <div className="flex w-64 items-center space-x-3 rounded-lg bg-white p-3 shadow-lg">
+    <div className="flex h-10 w-10 items-center justify-center rounded-full bg-primary text-lg font-semibold text-white">
+      {user.first_name[0]}
+    </div>
+    <div className="space-y-0">
+      <h3 className="text-sm font-semibold text-gray-800">
+        {user.first_name} {user.last_name}
+      </h3>
+      <p className="text-xs text-gray-500">@{user.username}</p>
+      <p className="text-xs text-gray-500">{user.user_type}</p>
+    </div>
+  </div>
+);
+
+const MarkdownPreview: React.FC<{ markdown: string }> = ({ markdown }) => {
+  const [userCache, setUserCache] = useState<Record<string, UserModel>>({});
+  const facilityId = "81092ced-8720-44cb-b4c5-3f0ad0540153";
```

Comment:
I assume this was hardcoded for testing purposes. Shouldn't be there in final version.

Reply:
will change

Reply:
yupp